Tests/Model/SPITransactionTests.m

```diff
@@ -150,6 +184,15 @@ - (void)testInitiateCashOutOnlyRequest {
     }];
 }
 
+- (void)testInitiateCashOutOnlyRequestWithSurcharge {
+    SPIClient *client = [SPITestUtils clientWithTestSecrets];
+    
+    client.state.status = SPIStatusPairedConnected;
+    [client initiateCashoutOnlyTx:@"test" amountCents:10 surchargeAmount:1 completion:^(SPIInitiateTxResult *result) {
+        XCTAssertNotNil(result);
```

Comment:
Again, use XCTestExpectation: https://developer.apple.com/documentation/xctest/asynchronous_tests_and_expectations/testing_asynchronous_operations_with_expectations

Reply:
This test is about Transactions via websocket. Previous test methods were developed like that.